apps/new-dealer/controllers/storage-address-lookup.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+const BaseController = require('hof').controllers.base;
+const ErrorController = require('hof').controllers.error;
+const _ = require('lodash');
+
+module.exports = class StorageAddressLookup extends BaseController {
+  locals(req, res) {
+    const locals = super.locals(req, res);
```

Comment:
very minor gripe but maybe group const and let together when they're this close?